src/sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,18 @@
+// COLOR VARIABLES
```

Comment:
You can also choose a color palatte for your app from places like:

https://colorhunt.co/